lib/stream.js

```diff
@@ -1,11 +1,18 @@
 'use strict';
 
-module.exports = Stream;
-
-const EE = require('events');
 const util = require('util');
 
-util.inherits(Stream, EE);
+function Stream() {
+  EE.call(this);
+}
```

Comment:
What I meant previously is why can't we just move this into the new internal module and export it into this file?